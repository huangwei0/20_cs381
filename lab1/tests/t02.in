debug off
load a_tracks.txt
filter title
count tracks
count words
count characters
