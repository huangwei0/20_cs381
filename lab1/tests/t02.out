54586
186792
1007462
